### ISSUE TITLE
Attend CodeDay talks/workshops

### ISSUE THREAD

**Issue by mikelbrierly** [developer-delta/halogen#52]
## User Story:

> As a developer,
I want to attend weekly workshops,
So that I can learn and help to advance my career

---

## Acceptance Criteria:

- [x] Attend one CodeDay talk or workshop
  - Add a comment to this story mentioning the name of the workshop, and something interesting you learned 
- [x] Attend a second CodeDay talk or workshop
  - Add a comment to this story mentioning the name of the workshop, and something interesting you learned

**Comment by Hanna-N9** [comment]
I attended a Codeday talk called "17 Tips for Working in Tech: almost everything I learned at Microsoft in 17 years" by Nicole Steinbok on July 15. I found it interesting that it is important not to lose vacation days because it would mean working for a company for free. Another thing is to embrace feedback without being annoyed or upset with it since it is better to step back and take time to think about it. I found this lesson to be most interesting compared to other lessons. "Like others first, they will like you. To be interesting, be interested." This made me want to reflect and how I should be interested in others. This is something that I want to keep in mind.

**Comment by Hanna-N9** [comment]
The name of the workshop was "Introduction to Robot Learning (ML and AI)" on July 20. I wasn't familiar with the topic besides hearing about it. I found it interesting that some of the first industrial robots were introduced in the 1950s that do not need human interactions and already have repeat pre-programmed. This differs from next-generation robots that require adaptation to changes and human-robot interactions, like programming a robot during trials. I learned that without machine learning, a person studying robots needs expert knowledge, predicts what would happen next, and do a lot of programming. Machine learning, however, makes those robots adaptive and self-reliant.  I found it interesting that creating robots or programming can be time-consuming and requires 1000+ trials. That made me have a change of perspective about it
